examples/experimental/torch/classification/configs/inception_v3_imagenet_int8.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
can't this be replaced by the use of https://github.com/openvinotoolkit/nncf/blob/develop/examples/torch/classification/configs/quantization/inception_v3_imagenet_int8.json?

Reply:
As mentioned in `README.md`, this configuration file was slightly edited to close the gap with `FracBits` configuration from `overflow_fix` and `batch_size`.